demisto_sdk/commands/create_id_set/create_id_set.py

```diff
@@ -14,6 +21,54 @@ def __init__(self, output: str = '', input: str = '', print_logs: bool = True):
         self.output = output
         self.input = input
         self.print_logs = print_logs
+        self.id_set = OrderedDict()  # type: ignore
 
     def create_id_set(self):
-        return re_create_id_set(id_set_path=self.output, pack_to_create=self.input, print_logs=self.print_logs)
+        self.id_set = re_create_id_set(id_set_path=self.output, pack_to_create=self.input, print_logs=self.print_logs)
+        self.modify_id_set_command_to_integration_of_playbook()
+        return self.id_set
+
+    def modify_id_set_command_to_integration_of_playbook(self):
+        """
+        Modifies playbook set in id_set dictionary once it was created, and finally saves it to a file.
+        Each playbook that has "command_to_integration" field will be modified :
+        - command name value will be a list of all integrations that implements this command (instead of use "" ).
+        """
+        command_name_to_implemented_integration_map = self.create_command_to_implemented_integration_map()
+
+        playbooks_list = self.id_set['playbooks']
+        for playbook_dict in playbooks_list:
+            playbook_name = list(playbook_dict.keys())[0]
+            playbook_data = playbook_dict[playbook_name]
+            commands_to_integration = playbook_data.get("command_to_integration", {})
+            for command in commands_to_integration:
+                is_command_implemented_in_integration = command in command_name_to_implemented_integration_map
+                if is_command_implemented_in_integration:
+                    implemented_integration = command_name_to_implemented_integration_map[command]
+                    commands_to_integration[command] = implemented_integration
+
+        self.save_id_set()
```

Comment:
I think this should happen in the `create_id_set`, don't you?
Contributes to the single responsibility principle